tests/integration/telemetry/stackdriver/stackdriver_filter_dry_run_test.go

```diff
@@ -67,7 +67,7 @@ func testDryRun(t *testing.T, policies []string, cases []dryRunCase) {
 									return err
 								}
 								return verifyAccessLog(t, cltInstance, tc.wantLog)
-							}, retry.Delay(framework.TelemetryRetryDelay), retry.Timeout(framework.TelemetryRetryTimeout))
+							}, retry.Delay(framework.TelemetryRetryDelay), retry.Timeout(framework.TelemetryRetryTimeout*2))
```

Comment:
The timeout is 80s. Any idea why it takes so long?

Reply:
It's the just existing default value and I just used it, not sure why it is set to 80s.

I think in most cases it does not need the whole 80s, usually should complete in 60s.

Reply:
Even 60s seems quite long.. Why authz policy would take so long to propogate?

Reply:
I am fine extending the time out, but just want to make sure that longer timeout is not masking some issues.

Reply:
I think the slowness is actually in the stackdriver instead of the authz policy propogation.

In the `TestStackdriverAuthzDryRun_Allow`, the proxy already has the expected policy (`ns[istio-echo-1-56410]-policy[dry-run-allow]-rule[0]`).

The test has 2 test cases using the above policy:
1. `TestStackdriverAuthzDryRun_Allow/matched` sends request that matches the policy. The test passed indicating the policy is already propagated.
2. `TestStackdriverAuthzDryRun_Allow/notMatched` sends request that does NOT match the policy. The test failed as it could not find the expected log entries. The generated [log entries](https://prow.istio.io/view/gs/istio-prow/logs/integ-telemetry-k8s-tests_istio_postsubmit/1400279435290087424#1:build-log.txt%3A2179) for HTTP requests are still from the `TestStackdriverAuthzDryRun_Allow/matched`. Interestingly, the log entries for gRPC requests are correct so it seems the slowness is only in HTTP.